ansible/group_vars/all/all

```diff
@@ -15,6 +15,23 @@ alaska_cloud: alaska
 alaska_homedir: /alaska
 alaska_softiron: 10.4.99.101
 
+# OpenStack fully qualified project name
+project_name: p3@default
+
+# Virtual IP address of the controller node
+controller_vip: 10.60.253.1
+
+# IP address from which the logging and monitoring server can be accessed
+alaska_monitoring_server: 10.60.253.3
+
+# OpenStack credentials for posting logs and metrics for the P3 project
+monasca_agent_p3_username: p3-monasca-agent
+monasca_agent_p3_password: "{{ vault_monasca_agent_password }}"
```

Comment:
Ah, now I get what you were referring to earlier with vault passwords.
On Alaska, see: openstack secret list
I prefer that approach because we don't have to pass around vault-encrypted data.
What do you think?

Reply:
Sounds good. You mentioned about looking at that at a later time - are you happy to merge this for now?